src/main/java/domain/Generator.java

```diff
@@ -0,0 +1,15 @@
+package domain;
+
+public class Generator {
+    private int MAX_LIMIT_NUMBER = 10; //default value
```

Comment:
사용하고 계신 대문자와 언더스코어의 조합 네이밍은 상수만을 위한 컨벤션입니다.

클래스 내에서 쓰임은 상수와 다를게 없긴 하지만 final로 지정하여 상수임을 더 명확히 하는 것이 좋지 않을까요?

```suggestion
    private static final int MAX_LIMIT_NUMBER = 10; //default value
```

또한 상수는 private static final을 사용하는 것이 일반적입니다. 
클래스 변수는 변경될 일이 없는 상수에 지정하여 메모리를 아낄 수 있습니다. 😄